source/_integrations/ffmpeg_noise.markdown

```diff
@@ -73,3 +73,18 @@ To experiment with values:
 ```bash
 ffmpeg -i YOUR_INPUT -vn -filter:a silencedetect=n=-30dB:d=1 -f null -
 ```
+
+### Troubleshooting
+
+#### Unresponsive after a while
+
+If the noise sensor becomes unresponsive, make sure you have `extra_arguments: -nostats` in the configuration, see [Issue 32](https://github.com/home-assistant-libs/ha-ffmpeg/issues/32#issuecomment-727569153) for details.  
+
```

Comment:
We should not link to issues. If the issue has content that is important, it should be added to the documentation directly.

```suggestion
If the noise sensor becomes unresponsive, make sure you have `extra_arguments: -nostats` in the configuration.  
```